### DISCUSSION TITLE
Getting the first articles right

### DISCUSSION THREAD

**Discussion by Lottamob** [thinkjrs/lottamob#19] [General]
I have uploaded pdfs under my github, research, for the three articles we have posted on my homepage. Wanna use these before I keep posting so we see that it turn out the way we want?

**Comment by thinkjrs** [comment]
Yes!

Here's a perma-link to the first one i'll use within the site: https://github.com/Lottamob/research/blob/d456691972843294c8af24e69610dae8624a7111/Coyne%20Moberg%202014%20The%20Political%20Economy%20of%20State-Provided%20Targeted%20Benefits%20Mercatus%20working%20paper.pdf

**Comment by thinkjrs** [comment]
Below is a short clip on how to get perma-links.  

Basic steps:

1. Navigate to the file you want a link to in Github
2. press `y` on your keyboard and the url will change
3. Copy-pasta that newfangled perma-link url! 

And you're done, drop tha 🎙️! 

![Kapture 2021-06-25 at 08 53 04](https://user-images.githubusercontent.com/16545607/123435134-d6c35880-d592-11eb-8573-7fe6e5bfbf38.gif)

**Reply by Lottamob** [reply]
Thanks! Should I use that link to include in the description on my homepage? 
I see, you will use those links to PDFs rather than linking to PDFs directly.
All my URLs for the uploaded articles already have decent permalinks, with some words rather than just numbers.

**Reply by Lottamob** [reply]
Uploaded a couple more articles. Let me know whether I am doing this right before I proceed. Thanks.

**Reply by thinkjrs** [reply]
Looks good -- Text me "hey redeploy the site" when you add any content to the CMS application. I've done that already so you have some new content up. 

I have to rebuild the static site (each time) so that these changes you add from the CMS are built into what users see. 

On another note, I added a couple of feature issues I plan to address this Friday/Sat.

**Comment by Lottamob** [comment]
Okay thanks!
